test/cli/commands_test.py

```diff
@@ -678,7 +678,7 @@ def test__cli__command__fix(rule, fname):
     with open(fname) as test_file:
         generic_roundtrip_test(test_file, rule)
 
-
+# Money maker
```

Comment:
I think this is a good test, but I'm not sure I would agree that it's the _best_ test. What's the thinking here?

Reply:
I honestly don't know how that word got there, I will remedy